### PR TITLE
[RunII_106X_v2] reproduce JetHT after Nov22 dCache issue

### PR DESCRIPTION
**Pull request by mneukum**
Reproduction of JetHT after Nov22 dCache issue for RunII_106X_v2.

Recovery task for UL18 RunD (~0.3% of the dataset) is still running and I will add all UL18 xml files once it's done

crab output was written to gridka and then copied to the group storage at DESY.

### CODE REVIEW

File: CrossSectionHelper.py [hunk 1]
```diff
@@ -608,7 +608,7 @@ class MCSampleValuesHelper():
             "NEvents" : NEventsValues(
                 NEVT_UL16preVFP=9726665+133752091,
                 NEVT_UL17=63043590,
-                NEVT_UL18=78255208,
+                NEVT_UL18=78253065,
```
Comment (finnlabe) [inline CrossSectionHelper.py:611]: Do we have some crashed jobs that explain this change in event counts?

**Comment by mneukum** [comment]
About the numbers in JetHT 2018A/B:

The numbers at the end of the xml files are the same as before the Nov22 reproduction.

Looking at the google spreadsheet, you can see that it changed from "Data (with MET filter fix)" to "(redone dCache) Data (with MET filter fix)", where the newer "_GT36" samples were used. I assume this was the reproduction in March?
Back then the CrossSectionHelper.py was not updated, which I included here.

I was also curious about this and looked into it. The difference is already in the DAS samples [1] but I didn't have time to understand why exactly that update changed the number of events and then admittedly forgot about it.... 

It's probably a good idea to follow up on this and document the reason here or in mattermost.

[1]
`dasgoclient --query="dataset=/JetHT/Run2018A-UL2018_MiniAODv2_GT36-v1/MINIAOD | grep dataset.nevents"`
171502033
`dasgoclient --query="dataset=/JetHT/Run2018A-UL2018_MiniAODv2-v1/MINIAOD | grep dataset.nevents"`
171484635

`dasgoclient --query="dataset=/JetHT/Run2018B-UL2018_MiniAODv2_GT36-v1/MINIAOD | grep dataset.nevents"`
78253065
`dasgoclient --query="dataset=/JetHT/Run2018B-UL2018_MiniAODv2-v1/MINIAOD | grep dataset.nevents"`
78255208

**Comment by mneukum** [comment]
@finnlabe 
ok, let me follow up on this really quick.

Here are two links related to this:
- [1] shows the dataset tiers from RAW upto NANOAOD with produced event fractions
- [2] is the PdmV twiki page for UL18 data

Apparently, the target is <0.1 permille accuracy. That also seems to be the reason why there's no "_GT36" reproduction for RunD

[1]
https://pdmv-pages.web.cern.ch/rereco_ul/full_table.html?dataset=%2FJetHT%2F*2018*&miniaod_dataset=MiniAODv2
[2]
https://twiki.cern.ch/twiki/bin/view/CMS/PdmVDatasetsUL2018

**Comment by finnlabe** [comment]
Thanks for these answers! Maybe I can invite @MatthiesC and @mschrode  to double-check here before we merge this?

**Comment by MatthiesC** [comment]
This looks good so far. We can merge (unless we want to wait for UL18 Run D to be part of this PR)

**Comment by finnlabe** [comment]
@apaasch needed this merged, I I'll do that now that you double-checked @MatthiesC . UL18 Run D can then be  a separate PR.

**Pull request merged**